test/house_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+
+  def setup
+    @house = House.new("$400000", "123 sugar lane")
+    @room_1 = Room.new(:bedroom, 10, '13')
+    @room_2 = Room.new(:bedroom, 11, '15')
+    @room_3 = Room.new(:living_room, 25, '15')
+    @room_4 = Room.new(:basement, 30, '41')
+  end
+
+  def test_it_exists
+    assert_instance_of House, @house
+  end
+
+  def test_house_attributes
+    assert_equal 400000, @house.price
+    assert_equal "123 sugar lane", @house.address
+    assert_equal [], @house.rooms
+  end
+
+  def test_house_can_add_rooms
+    assert_equal [], @house.rooms
+    @house.add_room(@room_1)
+    @house.add_room(@room_2)
+    assert_equal 2, @house.rooms.size
+    assert_instance_of Room, @house.rooms[0]
+    assert_instance_of Room, @house.rooms[1]
+    assert_equal :bedroom, @house.rooms[0].category
+  end
+
+  def test_if_house_is_above_market_average
+    refute @house.above_market_average?
```

Comment:
If you have multiple possible outcomes for a method, you should test all of them. I'd expect to see a second house here that is above market average, and the corresponding assertion.